File: C++/CElsiusFahre.cpp
/*
Escribe un programa que convierta una temperatura dada en Celsius a Fahrenheit y viceversa.
*/

#include <iostream>
using namespace std;
int main()
{
    float ingresaTemperatura;
    double celsiusFahre, FahrenheitCelsi;
    int opcion;

    cout << "1. Celsius a Fahrenheit." << endl;
    cout << "2. Fahrenheit a Celsius." << endl;
    cout << "Elige 1 o 2: ";
    cin >> opcion;

    switch (opcion)
    {
    case 1:
        cout << "Ingresa temperatura en Celsius: ";
        cin >> ingresaTemperatura;

        celsiusFahre = ((ingresaTemperatura * 9 / 5) + 32);
        cout << "Los " << ingresaTemperatura << " Celsius a " << celsiusFahre << " Fahrenheit";
        break;

    case 2:
        cout << "Ingresa temperatura en Fahrenheit: ";
        cin >> ingresaTemperatura;

        FahrenheitCelsi = ((ingresaTemperatura - 32) * 5 / 9);
        cout << "Los " << ingresaTemperatura << " Fahrenheit a " << FahrenheitCelsi << " Celsius";
    default:
        break;
    }
}

File: Python/CElsiusFahre.py
opcion = int(input("""1. Celsius a Fahrenheit.
2. Fahrenheit a Celsius.
Elige 1 o 2: """))

match (opcion):
    case 1:
        ingresaTemperatura = float(input("Ingresa temperatura en Celsius: "))
        celsiusFahre = ((ingresaTemperatura * 9/5) + 32)
        print (f"De {ingresaTemperatura} Celsius es igual a {celsiusFahre:.0f} grados Fahrenheit.")
    case 2:
        ingresaTemperatura = float(input("Ingresa temperatura en Fahrenheit: "))
        FahrenheitCelsi = round((ingresaTemperatura - 32) * 5/9)
        print (f"De {ingresaTemperatura} Fahrenheit a {FahrenheitCelsi:.0f} grados Celsius.")